ground/src/main/java/com/google/android/ground/ui/offlineareas/viewer/OfflineAreaViewerViewModel.kt

```diff
@@ -87,7 +86,7 @@ constructor(
   fun initialize(args: OfflineAreaViewerFragmentArgs) {
     offlineAreaId = args.offlineAreaId
     viewModelScope.launch(ioDispatcher) {
-      val thisArea = offlineAreaRepository.getOfflineArea(offlineAreaId).await()
+      val thisArea = offlineAreaRepository.getOfflineArea(offlineAreaId)!!
```

Comment:
Previously, await would return `null` if the `Maybe` was empty. Should we be doing the same here?

Reply:
It was actually `Single`, so await would throw an exception

Reply:
Sorry, you're right!